kraken/application_outage/actions.py

```diff
@@ -0,0 +1,55 @@
+import yaml
+import logging
+import time
+import kraken.cerberus.setup as cerberus
+from jinja2 import Template
+import kraken.invoke.command as runcommand
+
+
+# Reads the scenario config, applies and deletes a network policy to
+# block the traffic for the specified duration
+def run(scenarios_list, config, wait_duration):
+    failed_post_scenarios = ""
+    for app_outage_config in scenarios_list:
+        if len(app_outage_config) > 1:
+            with open(app_outage_config, "r") as f:
+                app_outage_config_yaml = yaml.full_load(f)
+                scenario_config = app_outage_config_yaml["application_outage"]
+                namespace = scenario_config["namespace"]
+                pod_selector = scenario_config["pod_selector"]
```

Comment:
Can we add in a default value here in case the user doesn't want to set a pod selector like '{}'

Reply:
Good idea, updated it. Thanks.

Reply:
Are we able to do it like the parameters in the main run_kraken file? This way if the pod_selector key isn't found in the file 
ex.) chaos_scenarios = config["kraken"].get("chaos_scenarios", [])

I'm getting the below when commenting out the pod_selector line in the scenario file

```
    pod_selector = scenario_config["pod_selector"]
KeyError: 'pod_selector'
```